.github/release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "0.0.1"
```

Comment:
Does it make sense to start out with the version `0.0.1`?
Feels like we should just start from v1.0.0

Reply:
Tested it en setting it to 0.0.1 makes the next/first release 0.1.0 (without `"bump-minor-pre-major": true,` I think it will be 1.0.0 have not tested that one) for 1.0 I prefer to have renamed flavour to flavor since it seems that is the spelling most ports use and it's already used in the documentations of this repo (text not examples)

Reply:
Alrighty, sounds good to make the v1 release the one with refactored variable names.